### DISCUSSION TITLE
Sudo writing rights to shared nfs volume

### DISCUSSION THREAD

**Discussion by tylerpotts** [nebari-dev/nebari#213] [Q&A] [answered]
**Be very careful when modifying files using the following method**

First get the name of the nfs pod:

```
$ k get pods -n dev
```

Copy the pod name thate has `nfs-server-nfs` in the name. Then run:

```
kubectl exec -n dev --stdin --tty <nfs-server-nfs-podname> -- /bin/bash
```

Now in the pod, run these commands to cd into the folder that holds all the user directories
```
cd /exports/home/
ls
```

Within this pod you have sudo rights to change any file for any user

**Comment by tylerpotts** [comment, marked as answer]
@jkellndorfer If you have k9s installed press the `s` key when on the `nfs-server` pod will skip all manual kubectl commands

**Comment by tylerpotts** [comment]
test github

**Comment by Adam-D-Lewis** [comment]
I'm closing old discussions in a repo cleanup effort.  Feel free to re-open if needed.
